EIPS/eip-1193.md

```diff
@@ -27,36 +27,19 @@ Promise resolves with `result` or rejects with `Error`.
 
 See the [available methods](https://github.com/ethereum/wiki/wiki/JSON-RPC#json-rpc-methods).
 
-#### eth_requestAccounts
-
-By default, the provider supplied to a new dapp has is a "read-only" provider with no accounts authenticated. See [EIP 1102: Opt-in account exposure](https://eips.ethereum.org/EIPS/eip-1102).
-
-To request accounts, call `ethereum.send('eth_requestAccounts')`. This will ask the user which account(s) they would like to authenticate to the dapp.
-
-Promise resolves with an array of the enabled account(s) addresses.
-
 ### Events
 
 Events are emitted using [EventEmitter](https://nodejs.org/api/events.html).
 
 #### notification
 
-All subscriptions from the node emit on "subscription type" (e.g. `eth_subscription`, or `ssh_subscription`). Attach listeners with:
+All subscriptions from the node emit on notification. Attach listeners with:
 
 ```js
-ethereum.on('eth_subscription', listener: (result: any) => void): this;
+ethereum.on('notification', listener: (result: any) => void): this;
 ```
 
-To create a subscription, call `ethereum.send('eth_subscribe')` or `ethereum.send('shh_subscribe')`. The subscription object will emit through the specific subscription type.
```

Comment:
I think this example below is super important so one can see how the notification object looks like. 

And as not the full object is returned, the reference to the wiki might not be enough.